src/components/NavBarComponent.jsx

```diff
@@ -31,6 +32,19 @@ export class NavBarComponent extends Component {
 
   toggleVisibility = () => this.setState({ visible: !this.state.visible });
 
+  navButton = () => {
+    const { visible } = this.state;
+
+    if (visible) {
+      return (
+        <Label className="nav-button">
+          Close <Icon name="delete" />
+        </Label>
+      );
+    }
+    return (<Icon id="hamburger" name="bars" />);
```

Comment:
For one liner JSX, there's no need for the parentheses. That is, the above logic can also be written as:
```
return <Icon id="hamburger" name="bars" />

Reply:
Noted. Rectifying it now